StackExchange.Redis/StackExchange/Redis/GeoUnit.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+
+namespace StackExchange.Redis
+{
+    /// <summary>
+    /// Units associated with Geo Commands
+    /// </summary>
+    public enum GeoUnit : short
+    {
+        /// <summary>
+        /// Meters
+        /// </summary>
+        m,
```

Comment:
Redis yelled at me when I changed it from this. It required the lowercase abbriviation for these. Perhaps that will change when it goes GA.